File: web/src/api/admin/gost_node_bind.js
import request from "../../request/index.js";

const baseUrl = '/v1/admin/gost/node/bind'

export const apiNormalGostNodeBindUpdate = (data) => {
    return request.request({
        url: `${baseUrl}/update`,
        method: 'POST',
        data
    })
}

File: server/router/admin/gost_node_bind.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/gost_node_bind"
	"server/global"
	"server/router/middleware"
)

func InitGostNodeBind(group *gin.RouterGroup) {
	g := group.Group("gost/node/bind", middleware.Auth(global.Jwt), middleware.AuthAdmin())
	g.POST("update", gost_node.Update)
}
